deliver/spec/app_screenshot_spec.rb

```diff
@@ -13,6 +14,29 @@ def screen_size_from(path)
 
   ScreenSize = Deliver::AppScreenshot::ScreenSize
 
+  describe "#initialize" do
+    context "when filename doesn't contain '(3rd generation)'" do
+      it "returns iPad Pro(12.9-inch)" do
+        screenshot = Deliver::AppScreenshot.new("path/to/screenshot/Screen-Name-iPad Pro (12.9-inch){2732x2048}.png", "de-DE")
+        expect(screenshot.screen_size).to eq(ScreenSize::IOS_IPAD_PRO)
+      end
+    end
+
+    context "when filename contains '(3rd generation)'" do
+      it "returns iPad Pro(12.9-inch) 3rd generation" do
+        screenshot = Deliver::AppScreenshot.new("path/to/screenshot/Screen-Name-iPad Pro (12.9-inch) (3rd generation){2732x2048}.png", "de-DE")
+        expect(screenshot.screen_size).to eq(ScreenSize::IOS_IPAD_PRO_12_9)
+      end
+    end
+
+    context "when filename contains 'ipadPro129'" do
+      it "returns iPad Pro(12.9-inch) 3rd generation" do
+        screenshot = Deliver::AppScreenshot.new("path/to/screenshot/ipadPro129-AAABBBCCCDDD{2732x2048}.png", "de-DE")
+        expect(screenshot.screen_size).to eq(ScreenSize::IOS_IPAD_PRO_12_9)
+      end
+    end
+  end
```

Comment:
Just a suggestion for the specs so that it matches the format of the other examples...

`when filename doesn't contain '(3rd generation)'` is already covered on line 58, in `should calculate all 12.9 inch iPad resolutions`.. I wonder if it would be good to add just a single example below that one like the following:

```ruby
it "should calculate all 12.9 inch iPad resolutions (3rd generation)" do
  expect_screen_size_from_file("iPad Pro (12.9-inch) (3rd generation)-01Name{2048x2732}.jpg").to eq(ScreenSize::IOS_IPAD_PRO_12_9)
  expect_screen_size_from_file("iPad Pro (12.9-inch) (3rd generation)-01Name-Landscape{2732x2048}.jpg").to eq(ScreenSize::IOS_IPAD_PRO_12_9)
  expect_screen_size_from_file("1_ipadPro129_1.Filename{2048x2732}.jpg").to eq(ScreenSize::IOS_IPAD_PRO_12_9)
  expect_screen_size_from_file("1_ipadPro129_1.Filename-Landscape{2732x2048}.jpg").to eq(ScreenSize::IOS_IPAD_PRO_12_9)
end
```

It'll also be good to add a similar example for the iMessage screenshots also (below line 130)

Reply:
I can change that

Reply:
This unfortunately did not make it into the PR. Could one of your create a follow up PR to improve these tests please? I would really appreciate that. Thanks!